paper_sections/04-examples.Rmd

```diff
@@ -109,11 +109,7 @@ Differences between models seem to be driven by the low seasonality of *Keratell
 #Getting the out of sample predictions for both models:
 
 #This function calculates the sum of squared deviances for out-of-sample data
-get_deviance = function(model,predicted, observed){
-  res <- model$family$dev.resids(observed, predicted, wt=1)
-  return(sum(res))
-}
-
+get_RMSE <- function(fit, obs) sqrt(mean((fit-obs)^2))
```

Comment:
I'm not sure what the best goodness of fit criterion is here. We originally were using MSE, which made sense when using a Gaussian distribution, but now that we've switched to a Gamma distribution, I'm not sure MSE or RMSE makes the most sense. Right now, for a couple species an intercept only model (with a random effect for taxon, but nothing else) actually gives lower RMSE than any of the smooth models, even when the smooth models seem to be accurately predicting the seasonal cycle out of sample. 

I would think that mean (or the sum of) deviances would be the preferred metric for GLMs, but it's a bit tougher to explain, and it doesn't have units they way that RMSE or MSE does. 

Thoughts?